content/applications/sales/sales/invoicing/down_payment.rst

```diff
@@ -222,6 +227,83 @@ At this point, the flow is now complete.
    order to confirm the delivery. Otherwise, the delivered quantity can be entered manually directly
    on the sales order.
 
+.. _sales/invoicing/down_payment/100-percent-down-payments:
+
+Example: request 100% down payment
+==================================
+
+The process of requesting a 100% down payment is similar to the process of setting up a :ref:`50%
+down payment <sales/invoicing/down_payment/50-percent-down-payments>`, but with fewer steps.
+
+.. note::
+   A 100% down payment is **not** the same as a full payment of the sales order.
+
+   A sales order paid through the regular invoice process will not allow any additional invoices to
+   be generated, and **will not** display the *Create invoice* button on the Sales Order.
```

Comment:
Since you are referencing the exact UI button, please match it's case :)
```suggestion
   be generated, and **will not** display the *Create Invoice* button on the Sales Order.
```